src/SrealityClient.js

```diff
@@ -6,21 +6,16 @@ const config = require('../config').sreality;
 const { errorAndDebug } = require('./utils/ErrorHandler');
 
 const PROPERTY_TYPES = {
-  ALL: {},
-  HOUSE: {
-    category_main_cb: 2,
-    category_type_cb: 1,
-  },
-  FLAT: {
-    category_main_cb: 1,
-    category_type_cb: 1,
-  },
-  PARCEL: {
-    category_main_cb: 3,
-    category_type_cb: 1,
-  },
+  FLAT: 1,
+  HOUSE: 2,
+  PARCEL: 3,
 };
 
+const TRANSACTION_TYPES = {
```

Comment:
Let's not duplicate code/constants and add and import from the config once it's there.